Cron_Philip/Assignments/interpreter/interpreter.js

```diff
@@ -0,0 +1,37 @@
+//problem 1:
+
+var first_variable;
+function firstFunc() {
+  first_variable = "Not anymore!!!";
+  console.log(first_variable);
+}
+console.log(first_variable);
+first_variable = "Yipee I was first!";
+console.log(first_variable);
+
+//problem 2:
+
+console.log('______________________________')
+
+var food;
+function eat() {
+  var food;
+  food = "half-chicken";
+  console.log(food);
+  food = "gone";       // CAREFUL!
+  console.log(food);
+}
+food = "Chicken";
+eat();
+console.log(food);
+
+//problem 3:
+
+console.log('______________________________')
+
+var new_word;
+function lastFunc() {
+  new_word = "old";
+}
+new_word = "NEW!";
+console.log(new_word);
```

Comment:
Where are you testing your functions in this?  Be sure to call them so you can see what they're doing.